examples/setup_run.py

```diff
@@ -127,34 +125,61 @@ def initial_sampler():
                                          initial_sampler=initial_sampler)
     return buffer
 
-
-def setup_trainer_and_run(cfg: DictConfig, setup_plotter: SetupPlotterFn,
+def get_load_checkpoint_dir(outer_checkpoint_dir):
```

Comment:
This ensures that the checkpoint with the most recent run, and most recent iteration is loaded.